### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by Mildred14**
- Add pages controller
- Add pages helper
- Add about us
- Add faq
- Add contact
- Add pages stylesheet

Before/ **After**
screenshot from 2018-12-06 18-11-16
screenshot from 2018-12-06 18-11-57

### CODE REVIEW

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+h1{
+    text-align: center;
+    color: green;
+}
```
Comment (AlejandroHdzM) [inline app/assets/stylesheets/pages.scss]: Always add a newline at the end of a file

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <div class="row">
+    <h1>About us</h1>
```
Comment (AlejandroHdzM) [inline app/views/pages/about.html.erb]: Internationalization here!

File: app/views/pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <div class="row">
+    <h1>About us</h1>  
+  </div>
+  <div class="row">
+      <div class="col s5">
+        <%= link_to image_tag('about-img.jpg', width: 200, height: 200)%>      
+      </div>
+      <div class="col s7">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
Comment (AlejandroHdzM) [inline app/views/pages/about.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
   <div class="nav-wrapper grey darken-4">
-    <%= link_to image_tag('finden.png', width: 150, height: 60), class: 'brand-logo' %>
+    <%= link_to image_tag('finden.png', width: 120, height: 60 ), class: 'brand-logo' %>
     <a href="#!" class="brand-logo"></a>
     <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons"> <%= t('header.menu') %> </i></a>
     <ul class="right hide-on-med-and-down">
-      <li><a href="sass.html"> <%= t('about_us') %> </a></li>
-      <li><a href="badges.html"> <%= t('header.fqa') %> </a></li>
-      <li><a href="collapsible.html"> <%= t('header.contact') %> </a></li>
+      <li><%= link_to "Acerca", pages_about_path %></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+      <li><%= link_to "Preguntas frecuentas", pages_faq_path %></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
+      <li><%= link_to "Contacto", pages_contact_path %></a></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
       <% if user_signed_in? %>
         <li><a>Logged in as <strong><%= current_user.email %></strong>.</li></a>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 8]
```diff
@@ -20,8 +20,8 @@
 </nav>
 
 <ul class="sidenav" id="mobile-demo">
-  <li><a href="sass.html">About us</a></li>
-  <li><a href="badges.html">FAQ</a></li>
+  <li><%= link_to "About", pages_about_path %></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 9]
```diff
@@ -20,8 +20,8 @@
...
+  <li><%= link_to "FAQ", pages_faq_path %></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 10]
```diff
@@ -20,8 +20,8 @@
...
   <li><a href="collapsible.html">Contact</a></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/views/welcome/_header.html.erb [hunk 11]
```diff
@@ -20,8 +20,8 @@
...
   <li><a href="mobile.html">Login</a></li>
```
Comment (AlejandroHdzM) [inline app/views/welcome/_header.html.erb]: And here!

File: app/controllers/pages_controller.rb [hunk 12]
```diff
@@ -0,0 +1,6 @@
+class PagesController < ApplicationController
+  def about
+  end
+  def faq
```
Comment (8geonirt) [inline app/controllers/pages_controller.rb]: It's a good practice to leave an empty line after `end` when declaring a new method.

File: app/views/welcome/_header.html.erb (hunk 5 above)
Comment (8geonirt) [inline app/views/welcome/_header.html.erb]: Fix that typo: `frecuentas`=> `frecuentes`

File: app/assets/stylesheets/pages.scss [hunk 13]
```diff
@@ -0,0 +1,7 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+h1{
```
Comment (diegosaul402) [inline app/assets/stylesheets/pages.scss]: Please add a space before the opening bracket

File: app/controllers/pages_controller.rb [hunk 14]
```diff
@@ -0,0 +1,8 @@
+class PagesController < ApplicationController
+  def about
+  end
+
+  def faq
+  end
+
+end
```
Comment (diegosaul402) [inline app/controllers/pages_controller.rb]: Please add a new line at the end of the file

File: app/views/welcome/_header.html.erb [hunk 15]
```diff
@@ -1,12 +1,12 @@
 <nav>
   <div class="nav-wrapper grey darken-4">
-    <%= link_to image_tag('finden.png', width: 150, height: 60), class: 'brand-logo' %>
+    <%= link_to image_tag('finden.png', width: 120, height: 60 ), class: 'brand-logo' %>
```
Comment (diegosaul402) [inline app/views/welcome/_header.html.erb]: This space after the height value is not required

File: app/views/pages/about.html.erb (hunk 2 above)
Comment (diegosaul402) [reply app/views/pages/about.html.erb]: You miss this change

File: app/assets/stylesheets/welcome.scss [hunk 16]
```diff
@@ -1,3 +1,52 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@font-face {
+  font-family: "Rubik";
+  src: url('/assets/Rubyk-Regular');
+  font-size: 30px;
+}
+
+body{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Add a space before `body`
It should be
```
body {
...
}
```

File: app/assets/stylesheets/welcome.scss [hunk 17]
```diff
@@ -1,3 +1,52 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@font-face {
+  font-family: "Rubik";
+  src: url('/assets/Rubyk-Regular');
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: I think you need to specify the extension of the file...
But I'd recommend you to get the font using Google fonts (https://fonts.google.com/specimen/Rubik), with that you can easily add all the fonts you need and they are also cached.

File: app/assets/stylesheets/welcome.scss [hunk 18]
```diff
@@ -1,3 +1,52 @@
...
+  font-size: 30px;
+}
+
+body{
+  font-family: "Rubik", sans-serif;
+}
+
+nav .nav-wrapper i, nav a.sidenav-trigger, nav a.sidenav-trigger i{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Add a space after the last`i`

File: app/assets/stylesheets/welcome.scss [hunk 19]
```diff
@@ -1,3 +1,52 @@
...
+  font-size: 33px;
+}
+
+nav .sidenav-trigger {
+  float: right;
+}
+.main-navigation-logo{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Space before `{`

File: app/assets/stylesheets/welcome.scss [hunk 20]
```diff
@@ -1,3 +1,52 @@
...
+  font-size: 64px;
+  margin-left: 15px;
+}
+.logo-nav{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Add a space before `{`

File: app/assets/stylesheets/welcome.scss [hunk 21]
```diff
@@ -1,3 +1,52 @@
...
+  width: 90px;
+}
+
+nav ul a{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Same thing about spaces

File: app/assets/stylesheets/welcome.scss [hunk 22]
```diff
@@ -1,3 +1,52 @@
...
+font-size: 16px;
+}
+
+nav ul a:hover{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: And here

File: app/assets/stylesheets/welcome.scss [hunk 23]
```diff
@@ -1,3 +1,52 @@
...
+  color: #00e25a;
+}
+
+.sidenav{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: And here

File: app/assets/stylesheets/welcome.scss [hunk 24]
```diff
@@ -1,3 +1,52 @@
...
+  background-color: #212121;
+}
+
+.sidenav li > a{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Also here 👀 💦

File: app/assets/stylesheets/welcome.scss [hunk 25]
```diff
@@ -1,3 +1,52 @@
...
+  color: white;
+  font-size: 16px;
+}
+
+.sidenav li > a:hover{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: And heeeereeeee ...👀 💦💦💦

File: app/assets/stylesheets/welcome.scss [hunk 26]
```diff
@@ -1,3 +1,52 @@
...
+  color: #00e25a;
+}
+
+.session-container{
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: 👀 💦

File: app/views/pages/about.html.erb [hunk 27]
```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <div class="row">
+    <h1><%= t('about_page.title')%></h1>  
+  </div>
+  <div class="row">
+      <div class="col s5">
```
Comment (8geonirt) [inline app/views/pages/about.html.erb]: Please fix this indentation

File: app/assets/stylesheets/welcome.scss [hunk 28]
```diff
@@ -1,3 +1,52 @@
...
+  font-family: "Rubik", sans-serif;
+}
+
+nav .nav-wrapper i, nav a.sidenav-trigger, nav a.sidenav-trigger i{
+  font-size: 33px;
+}
+
+nav .sidenav-trigger {
+  float: right;
+}
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Always leave an empty line after declaring a CSS rule, for example:
```
.my-class {
  ...
}

.another-class {
  ...
}
```

File: app/assets/stylesheets/pages.scss [hunk 29]
```diff
@@ -0,0 +1,8 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h1{
```
Comment (8geonirt) [inline app/assets/stylesheets/pages.scss]: @Mildred14 you haven't fix this thing that  @diegosaul402 mentioned

File: app/assets/stylesheets/welcome.scss [hunk 30]
```diff
@@ -1,3 +1,55 @@
...
+  margin-left: 15px;
+}
+
+.logo-nav {
+  width: 90px;
+}
+
+
+nav ul a {
+font-size: 16px;
```
Comment (8geonirt) [inline app/assets/stylesheets/welcome.scss]: Indent this property

**Pull request closed without merging**